store/siren-action-behavior.js

```diff
@@ -103,11 +103,16 @@ D2L.PolymerBehaviors.Siren.SirenActionBehaviorImpl = {
 			.then(function(resp) {
 				if (!resp.ok) {
 					var errMsg = resp.statusText + ' response executing ' + opts.method + ' on ' + href + '.';
-					return resp.json().then(function(data) {
-						throw { json: data, message: errMsg };
-					}, function(data) {
-						throw { string: data, message: errMsg };
-					});
+					const clone = resp.clone();
```

Comment:
Always cloning doesn't feel like a great solution, as I'm assuming that has to create a complete copy of the response object including all the data.

Reply:
This is one of 2 ways that I found to implement this (the old way is in the previous commits). I changed the implementation because @AlexBedley felt like the old implementation was suspicious. We should then agree to one solution (or create a new implementation).